daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
@huikang as explained in https://github.com/docker/docker/pull/15747#discussion_r37648103, this field has a very narrow meaning and I think we should deprecate it. If we can fix the networking assumptions in criu considering the above CNM design in libnetwork, then I think you wont find the need to depend on this as well.